ge/org.osate.ge.ba/src/org/osate/ge/ba/ui/properties/EditEmbeddedTextDialog.java

```diff
@@ -134,6 +153,7 @@ public void create() {
 	// Text modification listener that sets the OK button as enabled
 	// or disabled based on if the new text is valid
 	private ExtendedModifyListener getTextValidator(final BehaviorTransition behaviorTransition,
```

Comment:
Consider renaming to createTextValidator(). It creates something rather than just getting it.